### ISSUE TITLE
Indention wrong on multi-line instructions

### ISSUE THREAD

**Issue by himpich** [mrzapp/nextcloud-cookbook#226]
If I create multi-line instructions, after the second line break, the indention is not displayed in the recipe view (nor at the print). Example see screenshot.

![image](https://user-images.githubusercontent.com/6817957/82200543-6e219b00-98ff-11ea-9c65-49ca3030fa9e.png)

**Comment by mrzapp** [comment]
@himpich seems to me like step 3 isn't properly marked up, it's just bulleted with `*` characters. Can I see the JSON for this recipe (the `.json` file in the recipe folder)?

**Comment by himpich** [comment]
The bulleting with '* is just a habit, I don't try to do markdown. The indention is lost after the second linebreak, with or without the '*'

Example "instructions": 
```
Variationen:
100g Weisse Schokolade grob hacken und 100g Macadamia Nüsse - nicht zu fein hacken, ggf. einfach mit Hilfe zweier Küchenbretter kleiner Pressen. Gesalzene Nüsse vorher abwaschen.

Etwas Kakao in den Teig mischen. 100g weisse Schokolade grob hacken und 100g halbe Haselnüsse dazugeben

Ovomaltine in den Teig Mischen. Je mehr Ovomaltine man nimmt, umso fester wird der Teig und die Cookies am Ende knuspriger, da die Ovomaltine karamelisiert. Mit weisser Schokolade + Macadamia Nüssen anreichern.

50g grobe weisse Schokolade und 50g dunkle Schokolade mit Nüssen nach Wahl 

Die Hälfte des Teiges mit Kakao dunkel färben und mit weisser Schokolade versehen. Den hellen Teig mit dunkler Schokolade versehen. Nüsse nach Wahl. Jeweils eine kleinere Kugel formen und aneinanderpressen. Teig dann leicht auf das Blech drücken, ergibt "halb und halb" Cookies.

Alternativ zu gehackter Schokolade kann man auch Schokotropfen nehmen, diese wandern [jedoch] nicht so sehr in den Teig.

Addons: 
Cranberries, gehackt
Schokobons, halbiert
```

[recipe-without bulleting.json.txt](https://github.com/mrzapp/nextcloud-cookbook/files/4650053/recipe-without.bulleting.json.txt)

[recipe with bulleting.json.txt](https://github.com/mrzapp/nextcloud-cookbook/files/4650055/recipe.with.bulleting.json.txt)

**Comment by mrzapp** [comment]
@himpich this will be fixed with `v0.7.0`
